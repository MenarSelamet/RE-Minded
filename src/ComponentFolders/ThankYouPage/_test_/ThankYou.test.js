// import React from "react";
// import renderer from "react-test-renderer"
// import ThankYou from "../ThankYou";

// test('Thank You Page snapshot test', () => {

//     const component = renderer.create(<ThankYou />)

//     const tree = component.toJSON();

//     expect(tree).toMatchSnapshot();

// })
