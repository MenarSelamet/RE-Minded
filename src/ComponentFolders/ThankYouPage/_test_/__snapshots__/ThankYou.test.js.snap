// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thank You Page snapshot test 1`] = `
<div
  className="flex flex-col w-screen h-screen font-poppins text-black tracking-[-0.01em]"
>
  <h1
    className="absolute flex flex-wrap w-[300px] h-[75px] top-[156px] left-[84px]  text-3xl lg:text-5xl "
  >
    THANK YOU!
  </h1>
  <div>
    <div
      className="absolute flex flex-wrap w-[1273px] h-[36px] top-[251px] left-[84px] sm:text-lg leading-6 text-2xl lg:text-3xl leading-9 text-SubTexts"
    />
  </div>
  <div>
    <button
      className="bg-Buttons absolute w-[200px] md:[250px] lg:w-[300px] h-[60px] top-[343px] left-[84px]  rounded-md font-poppins text-2xl font-normal leading-9 tracking-normal text-center "
      type="button"
    >
      BACK TO HOME
    </button>
  </div>
</div>
`;
